### ISSUE TITLE
[Bug]: (icoextract) Glitched Icons

### ISSUE THREAD

**Issue by rjousse18** [bottlesdevs/Bottles#1587]
### Describe the bug

When I add desktop entry for Battle.net, I always have glitched icon like that :

![Capture d’écran du 2022-06-09 11-13-30](https://user-images.githubusercontent.com/31102194/172811691-a50fdb81-c5c7-40f4-9b8c-dca47d2685e6.png)

### To Reproduce

1. Go To Bottles
2. Add entry for Battle.net

### Package

Flatpak from Flathub

### Distribution

Fedora 36

### Health Check

```shell
Display:
    X.org: true
    X.org (port): :99.0
    Wayland: true
Graphics:
    vendors:
        amd:
            vendor: amd
            envs:
                DRI_PRIME: '1'
            icd: /usr/lib/x86_64-linux-gnu/GL/vulkan/icd.d/radeon_icd.x86_64.json:/usr/lib/i386-linux-gnu/GL/vulkan/icd.d/radeon_icd.i686.json
    prime:
        integrated: null
        discrete: null
Kernel:
    Type: Linux
    Version: 5.17.12-300.fc36.x86_64
Distro:
    Name: GNOME
    Version: '"42 (Flatpak runtime)"'
Disk:
    Total: 16829612032
    Free: 16829468672
RAM:
    MemTotal: 31.3GiB
    MemAvailable: 22.4GiB
Tools:
    cabextract: true
    p7zip: true
    patool: true
    glibc_min: '2.33'
Bottles_envs: null
```

### Display Protocol

Wayland

**Comment by mirkobrombin** [comment]
I'm able to replicate. Seems a icoextract problem.

**Comment by mirkobrombin** [comment]
This is strange, the icon is rendered correctly in chromium but not in the photo viewer or menus

**Comment by mirkobrombin** [comment]
fixed by cfa57d92e2cc9d12df8d877fc157f690341cebe5
